ui/src/Pages/Home/AccountPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { Container, Grid, Tab, Tabs } from "@mui/material";
+import { useState } from "react";
+import OrderedItems from "../../components/orderComponents/OrderedItems";
+import TrackingItems from "../../components/orderComponents/TrackingItems";
+import ReturnItems from "../../components/orderComponents/ReturnItems";
+
+function AccountPage() {
+    const [tabValue, setTabValue] = useState(0);
+
+    const handleTabChange = (event, newValue) => {
```

Comment:
you should add types to the arguments in the arrow function, for example:

`const handleTabChange = (event: string, newValue: string) => {''' function logic here '''}`

check out this for an explanation: https://chatgpt.com/share/72223e32-17a3-4099-bc36-71ab05f13dba